### ISSUE TITLE
Symphytum portable shut down in Combo Edit

### ISSUE THREAD

**Issue by pirol1972** [giowck/symphytum#129]
If edit Combo fields if changing something there is shown the white hand and in some circumstances there is a move of the field  over an other or not and then Symphytum stops and shut down
![symphytum-dh-shutdown](https://user-images.githubusercontent.com/55051414/74229502-cde62580-4cc2-11ea-95a8-840fab41215a.jpg)

**Comment by joshirio** [comment]
Yes I noticed that too since upgrading to Qt 5.12.7, will need to investigate why that happens now

**Comment by joshirio** [comment]
Was definitively caused by upgrade to Qt 5.12 and affects only Windows.

**Comment by joshirio** [comment]
As a temporary and quick fix, I reverted Qt back to version 5.11.3 for the windows installer and the portable zip. Please [download ](https://github.com/giowck/symphytum/releases) and install Symphytum again.

This will need a bigger investigation to understand what exactly was changed by Qt regarding to the combobox widget and its focus handling...

Leaving this issue open until it is properly resolved.
